### ISSUE TITLE
*Unrelated to this repo*

### ISSUE THREAD

**Issue by Bee-Mar** [KristjanESPERANTO/veggiekarte-local#9]
Hello,

I noticed you modified the layout of the MagicMirror 3rd Party modules page. I'm the creator of [MMPM](https://github.com/bee-mar/mmpm), which relies entirely on the layout of that page being stable.

**Comment by KristjanESPERANTO** [comment]
Hi @Bee-Mar,

thanks for reaching out! And thanks for MMPM! :slightly_smiling_face: 

I would like to do the restructuring of the page as I think it would be helpful for the users. Of course we can agree on the details.

From a rough look into your code, I think that the main problem is that I have changed the heading of the categories. You are looking for h3, but the categories are h2 with my change.

Is it enough to change `h3` to `h2` in [this line](https://github.com/Bee-Mar/mmpm/blob/f4323463dcd3a27d34044ee0adb1ae73c9ade9dd/mmpm/magicmirror/database.py#L54C9-L54C92)?

**Comment by KristjanESPERANTO** [comment]
I could also take back the heading change and keep h3. That makes no difference to the user and would perhaps be the simpler solution. What do you think?

**Comment by KristjanESPERANTO** [comment]
By the way: I also wanted to get in touch with you at some point. I am building an alternative module list and thought that there might be some synergy effects if we somehow connect MMPM and my list. I haven't thought about it yet in detail, but I wanted to take this opportunity to let you know. Here is the current status of the list: https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/

**Comment by Bee-Mar** [comment]
I agree that restructuring for the users is valuable. Keeping the categories as a h3 would be helpful and keep the changes made to parsing logic minimal as well. One other thing was changing the Author column to Maintainer. That's no problem at all and quickly be done on my end. Are there any other column names you wanted to change?

**Comment by KristjanESPERANTO** [comment]
No, I don't want to change more column names.

How do we proceed? My suggestion would be that I make my changes except for the two critical ones (Author->Maintainer + h3->h2). Then, when you have changed your module, you could make the Author->Maintainer change in the list yourself. Or do you have a better suggestion? :slightly_smiling_face:

**Comment by Bee-Mar** [comment]
> No, I don't want to change more column names.
> 
> 
> 
> How do we proceed? My suggestion would be that I make my changes except for the two critical ones (Author->Maintainer and h3->h2). Then, when you have changed your module, you could make the Author->Maintainer change in the list yourself. Or do you have a better suggestion? :slightly_smiling_face: 

I can copy the wiki to something locally that allows me to test the parsing for that. Is it alright if we delay the official change until either tonight or early tomorrow?

**Comment by KristjanESPERANTO** [comment]
I' parsing the wiki page as well and create a JSON file: https://github.com/KristjanESPERANTO/MagicMirror-3rd-Party-Modules/blob/main/docs/modules.json

But so far I haven't automated this yet, so my JSON file is not always as up-to-date as the wiki page. Accessing my JSON file now would have this disadvantage.

**Comment by KristjanESPERANTO** [comment]
> I can copy the wiki to something locally that allows me to test the parsing for that. Is it alright if we delay the official change until either tonight or early tomorrow?

Sure, no hurry! I don't have much time today either :slightly_smiling_face: Do your local tests and let me know when you are ready.

**Comment by Bee-Mar** [comment]
> > I can copy the wiki to something locally that allows me to test the parsing for that. Is it alright if we delay the official change until either tonight or early tomorrow?
> 
> 
> 
> Sure, no hurry! I don't have much time today either :slightly_smiling_face: Do your local tests and let me know when you are ready.

Thanks! I've also considered hosting a JSON file somewhere but it just ends up requiring me to chase more moving targets than I've already ended up with lol

**Comment by KristjanESPERANTO** [comment]
[Here](https://github.com/KristjanESPERANTO/veggiekarte-local/files/13794374/modules.md) is the file with the changes I suggested.

**Comment by Bee-Mar** [comment]
I'll start working with that and make the necessary changes. Thanks!

**Comment by Bee-Mar** [comment]
@KristjanESPERANTO Just providing an update. I haven't been able to get around to this just yet, but am aiming for this weekend since I'll also be making an additional bug fix

**Comment by KristjanESPERANTO** [comment]
I have done the restructuring again without the critical points. And I've just tested MMPM and it seems to work fine. The only thing I would like to change now is "Author" to "Maintainer" in the column headers.

**Comment by Bee-Mar** [comment]
@KristjanESPERANTO I've just revised the title of the "Extended plugins for MMM-GoogleAssistant". The introduction of a description under the name as well as hyperlinks within the category name causes MMPM to skip that category name entirely and uses an empty string for the category name.

Is there any chance we can delay the conversion of 'author' to 'maintainer'? The only reason I ask is since version 4.x.x recently came out, I would like to have people continue downloading the application and getting familiar with it before introducing a breaking change.

**Comment by KristjanESPERANTO** [comment]
No hurry. I'm basically quite satisfied with the current state for now. The change to maintainer can depend entirely on how it suits you :slightly_smiling_face:

**Comment by Bee-Mar** [comment]
@KristjanESPERANTO great, thanks! Grammatically, "maintainer" does make much more sense, and I'm for it in the long run. If we can do it in the beginning of February, I think it'll be pretty well established at that point.
